src/main/java/de/uni_tuebingen/ub/nppm/servlet/backend/AbstractBackendServlet.java

```diff
@@ -26,6 +27,19 @@ protected boolean isLoginRequired() {
         return true;
     }
 
+    protected boolean isAdminRequired(HttpServletRequest request, HttpServletResponse response) throws Exception {
+    Integer benutzerID = (Integer) request.getSession().getAttribute("BenutzerID");
+
+    if (benutzerID != null) {
+        Benutzer benutzer = BenutzerDB.getById(benutzerID);
+
+        if (benutzer != null) {
+            return benutzer.isAdmin();
```

Comment:
- The implementation of isAdminRequired in this class should only return a default value. All inheriting servlets that differ from this default value must override the function.
- The check if the current iser is an admin or not should be moved to the initRequest method, similar to the isLoginRequired mechanism.

Reply:
done look at commit: Administration